test/unit/test_helper.py

```diff
@@ -370,3 +371,44 @@ def test_get_export(self, verify_environment):
             get = helper.get_export(self.root_export, self.exports_file,
                                     export)
             self.assertEqual(get, '{"host": ["rw"]}')
+
+    @mock.patch('scality_manila_utils.helper.verify_environment')
+    def test_wipe_export_invalid(self, verify_environment):
+        export_name = 'export_with_grants'
+        host = '10.0.0.0/24'
+        export_point = os.path.join('/', export_name)
+        exports = ExportTable([
+            Export(
+                export_point=export_point,
+                clients={host: frozenset(['rw'])}
+            ),
+        ])
+
+        # Attempt to remove an export with existing grants
+        with mock.patch('scality_manila_utils.helper._get_defined_exports',
+                        return_value=exports):
+            with self.assertRaises(ExportHasGrantsException):
+                helper.wipe_export(self.root_export, self.exports_file,
+                                   export_name)
+
+        # Attempt to remove an export which does not exist
+        with self.assertRaises(ExportNotFoundException):
+            helper.wipe_export(self.root_export, self.exports_file, 'void')
+
+
+    @mock.patch('scality_manila_utils.helper.verify_environment')
+    def test_wipe_export(self, verify_environment):
+        export_name = "test_export"
+        absolute_export_path = os.path.join(self.nfs_root, export_name)
+        tombstone_path = os.path.join(self.nfs_root,
+                                      'TRASH-{0:s}'.format(export_name))
+
+        self.assertFalse(os.path.exists(absolute_export_path))
+        helper.add_export(self.root_export, export_name)
+        self.assertTrue(os.path.exists(absolute_export_path))
+
+        with mock.patch('scality_manila_utils.utils.fsync_path') as fsync_path:
```

Comment:
That should not pose a problem. The fsync makes sure that the block cache is flushed to the underlying storage. We will probably read from the buffer cache here, even with a real fsync.